packages/now-cli/src/commands/env/index.ts

```diff
@@ -18,16 +18,17 @@ import ls from './ls';
 import rm from './rm';
 
 const help = () => {
-  const placeholder = getEnvTargetPlaceholder();
+  const typePlaceholder = getEnvTargetPlaceholder();
```

Comment:
typo of `getEnvTypePlaceholder()`?

Reply:
Fixed in https://github.com/vercel/vercel/pull/5372/commits/f8180eddc9635def3bd89b4527e54c7d6065660b.